name.abuchen.portfolio/src/name/abuchen/portfolio/snapshot/filter/ClientSecurityFilter.java

```diff
@@ -178,6 +178,10 @@ private PortfolioTransaction convertToDelivery(PortfolioTransaction t, Portfolio
     private void convertTransfer(Function<Portfolio, ReadOnlyPortfolio> getPortfolio,
                     TransactionPair<PortfolioTransaction> pair)
     {
+        // in rare cases (wrong imports) the cross entry can be of type BuySellEntry
+        if (!(pair.getTransaction().getCrossEntry() instanceof PortfolioTransferEntry))
+            return;
+        
```

Comment:
I’d advise against hiding bad data, which might cause other issues down the line. When people see it failing, they can fix the transaction; the issue doesn’t seem to be widespread.

But +1 for fixing the importer.

Reply:
As mentioned in #1931, either let's write a ```Check``` that fixes the data or - if the user can fix it himself - then ask the user to fix it. Independently, we need to fix the importer.